batch_worker.py

```diff
@@ -135,7 +168,11 @@ def process_metric_data(self,start_date, end_date, tolerance,batch_size):
         df_merged["memory_delta"] = df_merged.memory_request - df_merged.memory_usage_max
         df_merged["cpu_recommendation"] =  df_merged.cpu_usage_max * (1 + tolerance / 100)
         df_merged["memory_recommendation"] = df_merged.memory_usage_max * (1 + tolerance / 100)
-        
+
+        df_merged["days_active"] = df_merged[
+            ["dummy_counter_x", "dummy_counter_y"]
+        ].fillna(0).apply(min, axis=1)
```

Comment:
@asamal4 Not clear on how we are getting values for dummy_counter_x and dummy_counter_y . Lets talk on this.